apps/examples/smartfs/read_optimization/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_SMARTFS_READ_OPTIMIZATION
+	bool "SmartFs Read Optimization Example"
```

Comment:
Could you rename this jSmartFS because journaling has been added?

Reply:
This thought came to my mind as well.
Since there are quite a few changes that have been done since we added journaling, we should consistently rename all of them as "jSmartFs" related changes.
So we should take care of all of them together.

Additionally, this name still remains under consideration as per this comment https://github.com/Samsung/TizenRT/pull/5289#pullrequestreview-809423667